### ISSUE TITLE
Assist Microphone - Unexpected error

### ISSUE THREAD

**Issue by bourliam** [rhasspy/hassio-addons#34]
Hi ! 👋 

I just tried to install and use the addon Assist Microphone but got an "Unexpected error"

Here are the logs I get when I start the addon :
```
ERROR:__main__:Unexpected error
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/homeassistant_satellite/__main__.py", line 200, in main
    async for _timestamp, event_type, event_data in stream(
  File "/usr/local/lib/python3.11/dist-packages/homeassistant_satellite/remote.py", line 28, in stream
    async with session.ws_connect(url) as websocket:
  File "/usr/local/lib/python3.11/dist-packages/aiohttp/client.py", line 1141, in __aenter__
    self._resp = await self._coro
                 ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/aiohttp/client.py", line 779, in _ws_connect
    resp = await self.request(
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/aiohttp/client.py", line 560, in _request
    await resp.start(conn)
  File "/usr/local/lib/python3.11/dist-packages/aiohttp/client_reqrep.py", line 899, in start
    message, payload = await protocol.read()  # type: ignore[union-attr]
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/aiohttp/streams.py", line 616, in read
    await self._waiter
aiohttp.client_exceptions.ServerDisconnectedError: Server disconnected
```

Nothing else and it just repeats this over and over.

I run HA on a RP4 with Home Assistant OS.

Happy to provide more information if needed as I am kind of stuck 😞 

Thanks for your awesome work

**Comment by xvlw** [comment]
I'm getting this also here : https://github.com/rhasspy/hassio-addons/issues/27

**Comment by synesthesiam** [comment]
Please upgrade to 2.2.1 and try again. I'm getting the port/protocol from the supervisor API now.

**Comment by bourliam** [comment]
Hi thanks for the updates, unfortunately it is still not working, I now have this error :
```
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'homeassistant'. (_ssl.c:992)
```
My installation is indeed not available with the host 'homeassistant'

**Comment by synesthesiam** [comment]
@bourliam Can you update to 2.2.2 and let me know if it's fixed? I'm using the IP address now from `core/info`

**Comment by tomlynn** [comment]
I'm on 2.2.3 and am still encountering the same error as bourliam.  Hostname mismatch, cert not valid for 'homeassistant'

**Comment by synesthesiam** [comment]
Do you have a self-signed certificate?

**Comment by tomlynn** [comment]
No, duckdns.  However, I seem to have solved my problem by filling in the
host field.  There is a suggestion below the field and since so many things
in home assistant require the FQDN:port, that's what I used.  I found after
several tries I had to omit the https and the port and then things began to
work with just servername.duckdns.org.  I think the help string in the
field should be very explicit about this.  After watching some youtube
videos I was left with the impression I should leave it blank, but then it
defaulted to homeassistant, which did not resolve.
